authors/settings.py

```diff
@@ -83,12 +84,18 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': os.environ['DATABASE'],
-        'USER':os.environ['USER'],
-        'HOST':os.environ['HOST'],
-        'PASSWORD':os.environ['PASSWORD']
+        'USER': os.environ['USER'],
+        'HOST': os.environ['HOST'],
+        'PASSWORD': os.environ['PASSWORD']
     }
 }
 
+EMAIL_USE_TLS = True
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_HOST_USER = 'cotuoma199@gmail.com'
```

Comment:
this email should also be in the env vars.

Reply:
awesome, and we will also use another email which is created for the app